microservicecloud/bundle-ssl/elasticsearch/elasticsearch.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDUjCCAjqgAwIBAgIUGr+NJMpgVN5QuI7iQRvi2t5XDX8wDQYJKoZIhvcNAQEL
```

Comment:
@TimothyHou Tim, just wanted to check, These are public key right?
@GoF-NUS-ISS/gof-core

Reply:
yes all public key